spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -194,12 +191,6 @@ module Oauth
       expect(response).to have_http_status :forbidden
     end
 
-    it "should not let a user edit his own application" do
-      controller.sign_in! user
-      get(:edit, params: { id: untrusted_application_user.id })
-      expect(response).to have_http_status :forbidden
-    end
```

Comment:
Put this spec back, change to success, and change title? (because a user can edit his app)

Reply:
Done